docs/advanced_topics/accessibility_considerations.md

```diff
@@ -126,6 +126,18 @@ Make sure to test your forms’ implementation with assistive technologies, and
 
 Here are things you can do to help authors create accessible content.
 
+### built-in accessibility checker
+
+Wagtail includes an accessibility checker built into the [`wagtailuserbar`](wagtailuserbar_tag). The checker can help authors create more accessible websites in compliance with the Web Content Accessibility Guidelines (WCAG) 2.0 and the Authoring Tool Accessibility Guidelines (ATAG) 2.0.
```

Comment:
We're talking about the user bar as a user-facing feature here (not just the template tag), so we should refer to it in plain English rather than formatting it as code:

```suggestion
Wagtail includes an accessibility checker built into the [user bar](wagtailuserbar_tag). The checker can help authors create more accessible websites in compliance with the Web Content Accessibility Guidelines (WCAG) 2.0 and the Authoring Tool Accessibility Guidelines (ATAG) 2.0.
```